moabb/datasets/bbci_eeg_fnirs.py

```diff
@@ -58,7 +58,33 @@ def fnirs_data_path(path, subject):
 
 
 class BBCIEEGfNIRS(BaseDataset):
-    """BBCI EEG fNIRS Motor Imagery dataset"""
+    """BBCI EEG fNIRS Motor Imagery dataset
+
+    Data Acquisition
+    ----------------------------------------
+    EEG and NIRS data was collected in an ordinary bright room. EEG data was recorded by a multichannel BrainAmp EEG amplifier with thirty active electrodes (Brain Products GmbH, Gilching, Germany) with linked mastoids reference at 1000 Hz sampling rate. The EEG amplifier was also used to measure the electrooculogram (EOG), electrocardiogram (ECG) and respiration with a piezo based breathing belt. Thirty EEG electrodes were placed on a custom-made stretchy fabric cap (EASYCAP GmbH, Herrsching am Ammersee, Germany) and placed according to the international 10-5 system (AFp1, AFp2, AFF1h, AFF2h, AFF5h, AFF6h, F3, F4, F7, F8, FCC3h, FCC4h, FCC5h, FCC6h, T7, T8, Cz, CCP3h, CCP4h, CCP5h, CCP6h, Pz, P3, P4, P7, P8, PPO1h, PPO2h, POO1, POO2 and Fz for ground electrode).
```

Comment:
FYI, Try to limit the line length at 79 char.
I'm soon gonna activate flake8 on travis, to enforce coding style, and this will make the test crash